Makefile

```diff
@@ -62,23 +62,19 @@ deps: gopath
 	env GOPATH=$(shell cd ../../../.. ; pwd) vndr
 
 .PHONY: validate
-validate:
+validate: install.tools
 	# Run gofmt on version 1.11 and higher
 ifneq ($(GO110),$(GOVERSION))
 	@./tests/validate/gofmt.sh
 endif
 	@./tests/validate/whitespace.sh
 	@./tests/validate/govet.sh
 	@./tests/validate/git-validation.sh
-	@./tests/validate/gometalinter.sh . cmd/buildah
+	./tests/tools/build/golangci-lint run
```

Comment:
I'd rather add golangci-lint as a separate PR.

Reply:
Sure, can do that.